docs/README.md

```diff
@@ -0,0 +1,28 @@
+# DBSP documentation
```

Comment:
how does this relate to all the RustDocs that we have?

Reply:
rustdocs are for API documentation, this is supposed to be documentation mainly for end-users (with some parts on how to contribute as well and deeper insights in how things work under-the-hood but that shouldn't be the main goal for now)

Reply:
I think rustdocs should be somewhere too. If they are published already we should at least link to them.
If we can put them here, even better.

Reply:
yes we'll link to docs.rs I think @blp is working on publishing it there